parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -225,6 +234,18 @@ private CompressionCodecName getCodec(TaskAttemptContext taskAttemptContext) {
     return CodecConfig.from(taskAttemptContext).getCodec();
   }
 
+  public static void setMaxPaddingSize(JobContext jobContext, int maxPaddingSize) {
+    setMaxPaddingSize(getConfiguration(jobContext), maxPaddingSize);
+  }
+
+  public static void setMaxPaddingSize(Configuration conf, int maxPaddingSize) {
```

Comment:
I don't think adding row group is more clear, this is the maximum amount of padding per block, not per row group. We just happen to check if we should pad between row groups. I guess it is a little strange because it ends up being the minimum row group size, but we could change the implementation so that isn't the case.